test_conformance/basic/test_explicit_s2v.cpp

```diff
@@ -186,6 +179,14 @@ int test_explicit_s2v_function(cl_device_id deviceID, cl_context context, cl_com
         {
             if( memcmp( convertedData, outPtr + destTypeSize * s, destTypeSize ) != 0 )
             {
+                if ((srcType == kHalf) && (destType == kFloat)
+                    && IsHalfNaN(*reinterpret_cast<cl_half *>(inPtr))
+                    && IsFloatNaN(*reinterpret_cast<cl_float *>(
+                        outPtr + destTypeSize * s)))
```

Comment:
This seems OK but it's a little odd to me that this is only needed for the half -> float case.  Couldn't this special-case be needed for any floating-point -> floating-point case, and especially the float -> double case?